include/gtk/Browser_window.hh

```diff
@@ -40,6 +40,7 @@ public:
   Browser_window(ebw_widget_t &ebw_widget);
 
   ready_signal_t &ready_signal() { return browser_ready_impl(); }
+// Dummy trigger for code review 2025-07-18

```

Comment:
I have named these terribly...
`inner` and `outer` definitely do not describe the intent. We need to find user-friendly names that suit all browser engine API categories.